packages/react-composites/src/composites/MeetingComposite/styles/MeetingCompositeStyles.ts

```diff
@@ -18,3 +18,24 @@ export const compositeOuterContainerStyles: IStackStyles = {
     flexWrap: 'wrap'
   }
 };
+
+/**
+ * @private
+ */
+export const NotificationIconStyles: IIconStyles = {
+  root: {
+    color: '#0078D4'
+  }
+};
+
+/**
+ * @private
+ */
+export const NotificationTextStyles: IStackStyles = {
+  root: {
+    position: 'absolute',
+    top: '0.1rem',
+    color: 'white',
```

Comment:
Should come from theme.

Reply:
resolved in this PR: https://github.com/Azure/communication-ui-library/pull/1378